code/drasil-example/ssp/lib/Drasil/SSP/TMods.hs

```diff
@@ -82,7 +81,7 @@ mcShrStrgth = tm (equationalModelU "mcShrSrgth" mcShrStrgthQD)
 
 ------------------------------------
 mcShrStrgthQD :: QDefinition
-mcShrStrgthQD = fromEqnSt' (shrStress ^. uid) (nounPhraseSP "Mohr-Coulumb shear strength")
+mcShrStrgthQD = fromEqnSt' (showUID shrStress) (nounPhraseSP "Mohr-Coulumb shear strength")
```

Comment:
Ah, no, we should never have `showUID` in examples. So this indicates a flaw somewhere. Maybe `fromEqnSt'` should take a `UID` ?